helpers/approveProcess.ts

```diff
@@ -0,0 +1,53 @@
+import { NextRouter } from "next/router";
+import { nullAsUndefined } from "null-as-undefined";
+import basePath from "./basePath";
+import getMatApiJwt from "./getMatApiJwt";
+import getProcessRef from "./getProcessRef";
+import isClient from "./isClient";
+
+const approveProcess = async (router: NextRouter): Promise<void> => {
```

Comment:
A bunch of this code is common to both approval and declining. Let's pull it out to be shared.